python/examples/01_tensorflow-basic.ipynb

```diff
@@ -0,0 +1,290 @@
+    "                                     input_names = input_names,\n",
+    "                                     output_names = output_names)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Configuring the server\n",
+    "\n",
+    "Now we configure the server. Here, specify the following:\n",
+    "- `input_data_type`: select a random port.\n",
+    "- `input_data_type`, `output_data_type`: Specify input and output data types as strings. \n",
+    "\n",
+    "<div class=\"alert alert-info\">\n",
+    "ℹ Accepted input and output data types are as follows: \n",
+    "    <ul>\n",
+    "        <li> Input: JSON, ARROW, IMAGE, ND4J (not yet implemented) and NUMPY. </li>\n",
+    "        <li> Output: NUMPY, JSON, ND4J (not yet implemented) and ARROW.</li>\n",
+    "    </ul>\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "port = np.random.randint(1000, 65535)\n",
+    "serving_config = ServingConfig(http_port = port,\n",
+    "                               input_data_type = 'NUMPY',\n",
+    "                               output_data_type = 'NUMPY')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `ServingConfig` has to be passed to `InferenceConfiguration`, in addition to the pipeline steps as a Python list. In this case, there is a single pipeline step: `tf_pipeline_step`. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "server = Server(config = InferenceConfiguration(serving_config = serving_config,\n",
+    "                                                pipeline_steps = [tf_pipeline_step]))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "By default, `Server()` looks for the Konduit Serving JAR `konduit.jar` in the directory the script is run in. To change this default, use the `jar_path` argument."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The configuration is stored as a dictionary. Note that the configuration can be converted to a dictionary using the `as_dict()` method: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "server.config.as_dict()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Configuring the client \n",
+    "\n",
+    "To configure the client, create a Client object with the following arguments: \n",
+    "- `input_names`: names of the input data\n",
+    "- `output_names`: names of the output data\n",
+    "- `input_type`: data type passed to the server for inference\n",
+    "- `endpoint_output_type`: data type returned by the server endpoint \n",
+    "- `return_output_type`: data type to be returned to the client\n",
```

Comment:
Also, mention that this can be used to convert the output returned from the server to the client into a different format. For example, NUMPY to JSON etc etc.